framework/packages/abstract-client/src/account.rs

```diff
@@ -317,25 +336,95 @@ impl<'a, Chain: CwEnv> AccountBuilder<'a, Chain> {
             install_modules,
             account_id: self.expected_local_account_id,
         };
-        let abstract_account = if let Some(owner_account) = self.owner_account {
-            owner_account
-                .abstr_account
-                .create_sub_account(account_details, Some(&funds))?
-        } else {
-            self.abstr.account_factory.create_new_account(
+        let abstract_account = match self.owner_account {
+            None => self.abstr.account_factory.create_new_account(
                 account_details,
                 ownership,
                 Some(&funds),
-            )?
+            )?,
+            Some(OwnerAccount::CurrentChain(owner_account)) => owner_account
+                .abstr_account
+                .create_sub_account(account_details, Some(&funds))?,
+            Some(OwnerAccount::RemoteChain(_)) => {
+                return Err(AbstractClientError::RemoteAccountOnBuild {})
+            }
         };
         Ok(Account::new(abstract_account, self.install_on_sub_account))
     }
+
+    /// Builds the [`RemoteAccount`].
+    /// Before using it you are supposed to wait Response.
+    /// For example: https://orchestrator.abstract.money/interchain/integrations/daemon.html?#analysis-usage
+    pub fn build_remote(&self) -> AbstractClientResult<(RemoteAccount<Chain>, Chain::Response)> {
```

Comment:
> Should we have this function return 2 objects ? I don't like it but I don't have a solution for it, because having the RemoteAccount is very handy

Something like `IbcPromise<T>` would be really nice to add on abstract_client! Because we know which calls are interchain

Reply:
Yes or because cw-orch-interchain is now public, we could just resolve the response call and return the created remote account

Reply:
> Yes or because cw-orch-interchain is now public, we could just resolve the response call and return the created remote account

Should I wait for next cw orch release before updating it?

Reply:
cw-orch-interchain is indepentent of cw-orch (interchain depends on cw-orch)